src/software/util/sml_fsm/dump_sml_fsm.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
```

Comment:
would be nice to have some more docs in this file

Reply:
I didn't mean to include this, whoops.